infrastructure/instances.tf

```diff
@@ -247,6 +247,8 @@ data "template_file" "nomad_client_script_smasher_smusher" {
 }
 
 resource "aws_instance" "smasher_instance" {
+  # This is an expense we don't need in the staging stack.
+  count = "${var.stage == "prod" || var.full_stack == "True" ? 1 : 0}"
```

Comment:
Why do we need to check for `var.stage == "prod"` here? `var.full_stack` should be `True` in prod right?

Reply:
I agree that it should be. However I don't think that we'll ever want prod to not have the full stack so this adds some protection against us ever accidentally changing that value in the `prod.environment` file.